libraries/UHS_host/macro_logic.h

```diff
@@ -141,7 +141,7 @@ AJK_IIF(AJK_BITAND(AJK_IS_COMPARABLE(x))(AJK_IS_COMPARABLE(y)) ) \
 #define AJK_MAKE_FUNS(AJK_v, AJK_args, AJK_count, AJK_body) AJK_EVAL(AJK_REPEAT(AJK_count, AJK_FUN, AJK_v, AJK_args, AJK_body))
 #if defined(AJK_TEST_MACRO_LOGIC)
 
-#define BODY(AJKindex) some(C, statement); contaning(a, test[AJKindex]);
```

Comment:
Rejected, Don't assume definition spellings within macros

Reply:
Rational: 

1. The macros don't actually generate usable code.
2. The output from this goes through my own private checking and depends on the misspelling, it was done on purpose.
3. Because no other code that is visible within the project uses it, avoid it. Besides, it isn't active, and has ifdef guards.